it/repositories/FileNotificationRepositoryISpec.scala

```diff
@@ -49,7 +51,7 @@ class FileNotificationRepositoryISpec extends IntegrationSpecCommonBase {
 
   "insertFileNotifications" should {
     "insert a single notification received from the backend" in new Setup{
-      val result: Boolean = await(repository.insertFileNotifications(Seq(notification1)))
+      val result: Boolean = await(service.insertNotificationRecordsIntoMongo(Seq(notification1)))
```

Comment:
You still want to test repository methods here, probably best if you create a service ISpec and test the service method separately i.e. converts seq of notifications to one record and check thats in mongo properly